app/controllers/clients_controller.rb

```diff
@@ -30,6 +30,10 @@ class ClientsController < ApplicationController
 
   def create
     clean_params = client_create_params
+    # Enforce DateCreated and DateUpdated
+    clean_params[:DateCreated] = Time.current
+    clean_params[:DateUpdated] = Time.current
```

Comment:
These should be getting set automatically from `save` below anyway, right?

Reply:
That was the root of the issue (they aren't saved automatically on the warehouse side).